core/src/main/resources/hudson/tasks/ArtifactArchiver/help-artifacts.html

```diff
@@ -6,3 +6,27 @@
    The base directory is <a href='ws/'>the workspace</a>.
    You can only archive files that are located in your workspace.
 </div>
+
+   <br>
+<div>
+   <p>Here are some examples of usage for declarative pipeline: </p>
```

Comment:
relevant to scripted pipeline as well:
```suggestion
   <p>Here are some examples of usage for pipeline: </p>
```